website/docs/users-sources/sources/social-logins/google/workspace/index.md

```diff
@@ -0,0 +1,197 @@
+
+Within the app creation page, fill in the following fields with your desired values:
+
+| Field       | Value                        |
+| ----------- | ---------------------------- |
+| Name        | authentik                    |
+| Description | Single Sign-On for authentik |
+
+### Google Identity Provider details
+
+Next, you'll be presented with information we'll need to configure authentik. Under _Option 2_, copy the SSO URL and download the certificate.
+
+:::warning
+authentik is acting as both a Service Provider (SP) to Google and an Identity Provider (idP) to your applications. Since we only need the SP configuration, you can ignore the Entity ID provided by Google.
+
+|           |                                                           |
+| --------- | --------------------------------------------------------- |
+| SSO URL   | `https://accounts.google.com/o/saml2/idp?idpid=#########` |
+| Entity ID | ~~`https://accounts.google.com/o/saml2?idpid=#########`~~ |
+
+:::
+
+### Service Provider details
+
+We'll need to provide Google with some information about our authentik instance, specifically the Assertion Consumer Service (ACS) URL. This URL is where Google will send the SAML response after a user has authenticated. We'll also need to provide the Entity ID, which can be any unique identifier, but we recommend using the URL of your authentik instance.
+
+| Field           | Value                                               |
+| --------------- | --------------------------------------------------- |
+| ACS URL         | `https://authentik.company/source/saml/google/acs/` |
+| Entity ID       | `https://authentik.company`                         |
+| Name ID format  | EMAIL                                               |
+| Name ID         | Basic Information › Primary Email                   |
+| Signed Response | Enabled ✅                                          |
+
+Keep these values handy in your text editor, as we'll need them later.
+
+### Attribute Mapping
+
+Next, we configure which user attributes Google should send to authentik.
+This is where we map the Google Directory attributes to the attributes that authentik expects.
+
+| Google Directory attributes       | App attributes |
+| --------------------------------- | -------------- |
+| Basic Information › Primary Email | `email`        |
+
+### Enable the app for your organization
+
+Finally, we complete the app creation process by saving the configuration.
+
+You should now see the new app in the list of SAML apps. View the app details and confirm that the SSO URL and Entity ID are correct. Note that you may need to **enable the app** for your organization to allow users to authenticate.
+
+---
+
+## authentik Configuration
+
+We'll now configure authentik to accept SAML authentication from Google Workspace.
+
+Start by logging into your authentik instance as an administrator and navigating to the _Admin Interface_.
```

Comment:
```suggestion
Start by logging into your authentik instance as an administrator and navigating to the Admin Interface.
```

Reply:
We don't do any special formatting for the Admin interface, since it is not really a UI component or element... it is the UI. ;-)